### DISCUSSION TITLE
马自达3昂克赛拉2017运动版自行加装CarPlay记录 | LEO'S NOTES

### DISCUSSION THREAD

**Discussion by giscus** [imleowoo/imleowoo.github.io#73] [General]
# 马自达3昂克赛拉2017运动版自行加装CarPlay记录 | LEO'S NOTES

马自达3昂克赛拉2017运动版自行加装CarPlay记录 - LEO'S NOTES

https://imleowoo.github.io/post/%E9%A9%AC%E8%87%AA%E8%BE%BE3%E6%98%82%E5%85%8B%E8%B5%9B%E6%8B%892017%E8%BF%90%E5%8A%A8%E7%89%88%E8%87%AA%E8%A1%8C%E5%8A%A0%E8%A3%85carplay%E8%AE%B0%E5%BD%95/

**Comment by imleowoo** [comment]
博主放一下昂克赛拉CarPlay的固件
